### ISSUE TITLE
Disable secret/private URL function

### ISSUE THREAD

**Issue by kcspamsplat** [cydrobolt/polr#137]
The way this functionality is presented, the shortened URL is a link/password pair. This is essentially security through obscurity. The feature request is to be able to disable this option so users are not lulled into a false sense of security.  Ultimately, for most uses, the target URL should implement security properly.

**Comment by cydrobolt** [comment]
I don't agree that we should disable this option. This setting isn't designed to "fool" anyone into a false sense of security -- rather, it presents an option that allows people to generate short links that are more difficult to guess. 

Without this option, all links generated would be easily guessable, even when that behavior is not desired. While it doesn't "protect" a link, per se, it helps prevent unauthorized people from guessing the link simply by counting up.

**Comment by kcspamsplat** [comment]
The request was for the ability to disable it as a configuration choice; not to eliminate the capability. So for those who think it is useful, the interface would continue displaying the option.

The false sense of security is that having a password on the shortened URL doesn't ensure that the target URL implements security, and if the target does indeed implement proper security, why bother having a password on the shortened URL?

**Comment by cydrobolt** [comment]
@kcspamsplat ah, okay. I'll change the ticket to make it more clear that this should be an option.
